bubblewrap.c

```diff
@@ -408,17 +421,16 @@ acquire_caps (void)
       /* Drop root uid, but retain the required permitted caps */
       if (setuid (getuid ()) < 0)
         die_with_error ("unable to drop privs");
+      did_setuid = TRUE;
```

Comment:
This approach means the file-caps based version of bwrap doesn't work. Although honestly, thats kinda usless vs setuid as it needs cap_sysadmin anyway.